dashboard/app/helpers/certificates_helper.rb

```diff
@@ -16,6 +16,14 @@ def certificate_image_url(name, course, donor)
     "/certificate_images/#{encoded}.jpg"
   end
 
```

Comment:
I would prefer to de-dup constants with the above function, but I wanted to minimize all changes to existing functionality. This PR is purely additive

Reply:
Added similar comments before refreshing the page 🙃